test/buidler-project/.solhint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "solhint:all"
```

Comment:
Doesn't this make the tests unstable? They are checking the number of reports, but we can't be sure that it won't change in future versions. New violations may appear due to new rules added into solhint.